airflow/models/dagbag.py

```diff
@@ -389,24 +388,19 @@ def collect_dags(
                 td = td.total_seconds() + (
                     float(td.microseconds) / 1000000)
                 stats.append(FileLoadStat(
-                    filepath.replace(dag_folder, ''),
+                    filepath.replace(settings.DAGS_FOLDER, ''),
```

Comment:
@milton0825 This was causing the parse time to metric to be emmited as `dag.loading-duration.`. Whoops :D

Reply:
.....